CryptoPkg/Library/TlsLibMbedtls/TlsConfig.c

```diff
@@ -921,7 +927,9 @@ TlsSetEcCurve (
   )
 {
   TLS_CONNECTION  *TlsConn;
-  mbedtls_ecp_group_id grp_id;
+  UINT16  *GroupList;
+
+  GroupList = AllocateZeroPool(sizeof(UINT16) * 2);

```

Comment:
Where to free this pool?  I am worried about potential memory leak.

Reply:
1. conf and fd context not free after use too: TlsConn->fd;TlsConn->Conf
2. if need to use pools, it is best to add them to Tls context.